generated/nidaqmx/nidaqmx_library.h

```diff
@@ -147,7 +147,7 @@ class NiDAQmxLibrary : public nidaqmx_grpc::NiDAQmxLibraryInterface {
+  int32 GetAnalogPowerUpStates(const char deviceName[], const char channelName[], float64* state, int32 channelType);
```

Comment:
This is a regression. These are `execl`-style varargs functions, so they require passing each array element as a separate parameter, plus a nullptr to terminate the list of varargs. Grpc-device handles these functions by generating the library method with a large, fixed number of parameters and then using functions like `get_lines_if` to fill all of the parameters with either valid values or nullptr.

Reply:
Sure. I have reverted those changes by removing `proto_only` field for those parameters in scrapigen in this PR.
https://github.com/ni/grpc-device-scrapigen/pull/169

Reply:
Much better, thanks.